src/gallium/drivers/tegra/tegra_context.c

```diff
@@ -45,16 +45,12 @@ static int tegra_channel_create(struct tegra_context *context,
 static void tegra_channel_delete(struct tegra_channel *channel)
 {
 	tegra_stream_destroy(&channel->stream);
+	drm_tegra_channel_close(channel->stream.channel);
 	FREE(channel);
 }
 
 static void tegra_channel_flush(struct tegra_channel *channel)
 {
-	int err;
-
-	err = tegra_stream_flush(&channel->stream);
-	if (err < 0)
-		fprintf(stderr, "tegra_stream_flush() failed: %d\n", err);
 }
```

Comment:
No, it is in use https://github.com/grate-driver/mesa/pull/4/files/99f982507084ce0d2d24a54ec455430cf4b218b8#diff-cf6b1c4d872eacea5a87b7af634f403dL490, the "flush" performs the job submission. Currently we need to "flush" the job right after the "stream ending", I removed the flushes in https://github.com/grate-driver/mesa/commit/947344446cbf0b40a30e07f33654e5a810c4764a and it was a mistake.